net/zmap/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.0
+PortGroup           github 1.0
+
+name                zmap
+github.setup        zmap zmap 3.0.0-beta1 v
+
+categories          net
+platforms           darwin
```

Comment:
`platforms darwin` is now implicit, so this line can be omitted.

Reply:
This tests OK locally as well.